courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<p>to explain why we are using virtual epochs, link to <a href="https://medium.com/google-cloud/ml-design-pattern-3-virtual-epochs-f842296de730" target="_blank">https://medium.com/google-cloud/ml-design-pattern-3-virtual-epochs-f842296de730</a>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done!</p>